.github/workflows/main.yml

```diff
@@ -178,7 +178,7 @@ jobs:
     - name: Check out the Unbundled AAOS Repo
       run: |
         HOST=https://android.googlesource.com/platform/superproject
-        BRANCH=ub-automotive-master-20230303
+        BRANCH=ub-automotive-master
```

Comment:
Do we want to just use master, or do we want to pin to a particular release again? (This looks like the most recent: https://android.googlesource.com/platform/manifest/+/refs/tags/ub-automotive-master-20230622 )

Reply:
Given that master is failing to build here (though it builds on my machine with a sync from yesterday), I'll give this a try

Reply:
Sounds good -- looks like the CI checks passed using the tag, too.